.changeset/pre.json

```diff
@@ -1,5 +1,5 @@
 {
-  "mode": "pre",
+  "mode": "exit",
```

Comment:
exiting `pre` mode now! Confirmed manually that all non-`astro` packages will just be changed from:

```
'@astrojs/solid-js': ^0.0.4-beta.0`
'@astrojs/solid-js': ^0.0.4`
```